### PR TITLE
Review help strings

### PR DESCRIPTION
**Pull request by jrodewig**
Ticket(s): [DOCS-3864](https://faunadb.atlassian.net/browse/DOCS-3864)

## Problem

Some of the various help strings and examples could use more detail and formatting.

## Solution

Adds detail and formatting to help strings and examples.

## Testing

Linted + reviewed using `./src/user-entrypoint.mjs <command>`.

[DOCS-3864]: https://faunadb.atlassian.net/browse/DOCS-3864?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by henryfauna — APPROVED** [review]

File: src/commands/database/create.mjs [hunk 1]
```diff
@@ -39,28 +39,28 @@ function buildCreateCommand(yargs) {
       name: {
         type: "string",
         required: true,
-        description: "the name of the database to create",
+        description: "Name of the database to create.",
       },
       typechecked: {
         type: "string",
-        description: "enable typechecking for the database",
+        description: "Enable typechecking for the database. Defaults to the typechecking setting of the parent database.",
```
Comment (echo-bravo-yahoo) [inline src/commands/database/create.mjs:46]: what does this do for a top-level db? is that worth mentioning?

File: src/commands/schema/commit.mjs [hunk 2]
```diff
@@ -73,12 +73,12 @@ function buildCommitCommand(yargs) {
...
     .example([["$0 schema commit"]])
     .version(false)
-    .help("help", "show help");
+    .help("help", "Show help.");
 }
 
 export default {
   command: "commit",
-  description: "Push the current project's .fsl files to Fauna.",
+  description: "Apply staged schema files to a database.",
```
Comment (echo-bravo-yahoo) [inline src/commands/schema/commit.mjs:81]: Are schema changes scoped to a db? Or wider than that?

File: src/commands/schema/commit.mjs [hunk 3]
```diff
@@ -73,12 +73,12 @@ function buildCommitCommand(yargs) {
     })
     .example([["$0 schema commit"]])
     .version(false)
-    .help("help", "show help");
+    .help("help", "Show help.");
```
Comment (echo-bravo-yahoo) [inline src/commands/schema/commit.mjs:76]: I can't comment on the correct line (it's context/not changed), but the string on line 69 of this file should end with a period:
```
description: "Prompt for user input (e.g., confirmations)",
```

File: src/lib/command-helpers.mjs [hunk 4]
```diff
@@ -4,16 +4,16 @@
...
   },
   url: {
     type: "string",
-    description: "the Fauna URL to query",
+    description: "URL for Fauna Core HTTP API requests made by the command. Defaults to https://db.fauna.com.",
   },
   secret: {
     type: "string",
-    description: "the secret to use when calling Fauna",
+    description: "Authentication secret for Fauna Core HTTP API requests made by the command. Mutually exclusive with `--database`.",
```
Comment (echo-bravo-yahoo) [inline src/lib/command-helpers.mjs:16]:
```suggestion
    description: "Authentication secret for Fauna Core HTTP API requests made by the command. Mutually exclusive with `--database` and `--role`.",
```

File: src/cli.mjs [hunk 5]
```diff
@@ -116,54 +116,54 @@ function buildYargs(argvInput) {
     .options({
       color: {
         description:
-          "whether or not to emit escape codes for multi-color terminal output.",
+          "Enable color formatting for the output. Uses ANSI escape codes.",
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply src/cli.mjs]: Thanks! Updated to cover the default and `--no-color`/`--color=false` with [`80db274` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/80db27480ccfa5e2ca86c20fe26aa393dea1329a).

File: src/cli.mjs [hunk 6]
```diff
@@ -116,54 +116,54 @@ function buildYargs(argvInput) {
...
         description:
-          "the profile in your config file to fetch CLI settings from",
+          "Profile from the CLI config file to use. Each profile specifies a set of CLI settings.",
         default: "default",
       },
       user: {
         alias: "u",
         type: "string",
-        description: "a user profile",
+        description: "User profile used to run the command. Create user profiles using `fauna login`.",
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply src/cli.mjs]: Good call. Updated with [`80db274` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/80db27480ccfa5e2ca86c20fe26aa393dea1329a).

File: src/cli.mjs [hunk 7]
```diff
@@ -116,54 +116,54 @@ function buildYargs(argvInput) {
...
       json: {
         type: "boolean",
-        description: "output the result as JSON",
+        description: "Output the results as JSON.",
         default: false,
       },
       quiet: {
         type: "boolean",
-        description: "only emit output",
+        description: "Only output the results of the command.",
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply src/cli.mjs]: Added with [`80db274` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/80db27480ccfa5e2ca86c20fe26aa393dea1329a).

File: src/cli.mjs [hunk 8]
```diff
@@ -116,54 +116,54 @@ function buildYargs(argvInput) {
...
         default: false,
       },
       verboseComponent: {
         description:
-          "components to emit diagnostic logs for; this takes precedence over the 'verbosity' flag",
+          "Comma-separated list of components to emit diagnostic logs for. Takes precedence over the `--verbosity` flag.",
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply src/cli.mjs]: Thanks. I documented both options with [`80db274` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/80db27480ccfa5e2ca86c20fe26aa393dea1329a).

File: src/commands/schema/commit.mjs (hunk 3 above)
Comment (jrodewig) [reply src/commands/schema/commit.mjs:76]: Updated with [`80db274` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/80db27480ccfa5e2ca86c20fe26aa393dea1329a) and [`a257cb8` (#451)](https://github.com/fauna/fauna-shell/pull/451/commits/a257cb8ddfbc1003d0ccf0af2a5cc772f2c361a4)

**Comment by jrodewig** [comment]
Merging. I'll take another pass once we get closer to release.

**Pull request merged**